### PR TITLE
PT-1936: Tooltip for toolbar command button

### PR DESCRIPTION
**Pull request by ErickLipin**
https://virtocommerce.atlassian.net/browse/PT-1934

### CODE REVIEW

**Review by eMazeika — APPROVED** [review]

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]
1. Add eng/ru localization for this examples;
2. Change key for title to `platform.titles.commands.refresh`.

File: docs/fundamentals/extensibility/blade-toolbar.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ $scope.blade.toolbarCommands = [
   {
     name: "Save",
     icon: 'fa fa-save',
+    title: 'Save action tooltip text'
```
Comment (Vectorfield4) [inline docs/fundamentals/extensibility/blade-toolbar.md:20]:
```suggestion
    title: 'Save action tooltip text' // or you can use keys for translation "platform.commands.titles.save",
```

**Review by Vectorfield4 — APPROVED** [review]

File: docs/fundamentals/extensibility/blade-toolbar.md [hunk 2]
```diff
@@ -28,6 +29,7 @@ $scope.blade.toolbarCommands = [
   {
     name: "Reset",
     icon: 'fa fa-undo',
+    title: 'Reset action tooltip text'
```
Comment (Vectorfield4) [inline docs/fundamentals/extensibility/blade-toolbar.md:32]:
```suggestion
    title: 'Reset action tooltip text',
```

File: docs/fundamentals/extensibility/blade-toolbar.md [hunk 3]
```diff
@@ -40,6 +42,7 @@ $scope.blade.toolbarCommands = [
   {
     name: "Change password",
     icon: 'fa fa-refresh',
+    title: 'Change password action tooltip text'
```
Comment (Vectorfield4) [inline docs/fundamentals/extensibility/blade-toolbar.md:45]:
```suggestion
    title: 'Change password action tooltip text',
```

**Review by Vectorfield4 — APPROVED** [review]

File: docs/fundamentals/extensibility/blade-toolbar.md (hunk 2 above)
Comment (ErickLipin) [reply docs/fundamentals/extensibility/blade-toolbar.md]: https://github.com/VirtoCommerce/vc-platform/pull/2298/commits/0b7df740bddb5952290e5d4aef58968c48120044

File: docs/fundamentals/extensibility/blade-toolbar.md (hunk 3 above)
Comment (ErickLipin) [reply docs/fundamentals/extensibility/blade-toolbar.md]: https://github.com/VirtoCommerce/vc-platform/pull/2298/commits/0b7df740bddb5952290e5d4aef58968c48120044

**Pull request merged**